package.json

```diff
@@ -9,14 +9,15 @@
     "prop-types": "^15.8.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-scripts": "5.0.1",
+    "react-scripts": "^5.0.1",
     "web-vitals": "^2.1.4"
   },
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "cypress:open": "cypress open"
```

Comment:
These scripts are used sometimes as a shortcut - as a developer there is little difference in my typing `npm run cypress:open` and `npm cypress open`. See how all the other scripts significantly shorten the commands needed to run the start, build, test, etc?